.gitmodules

```diff
@@ -23,3 +19,6 @@
 [submodule "external_libraries/link"]
 	path = external_libraries/link
 	url = https://github.com/Ableton/link
+[submodule "external_libraries/portaudio"]
+	path = external_libraries/portaudio/portaudio_library
```

Comment:
@brianlheim what do you think about this directory names? `portaudio` as top level and `portaudio_library` for the actual PA repo? I go back and forth on this, almost thinking that it could be the other way around (`portaudio_library` or `portaudio_something` as top level directory, and `portaudio` for actual PA repo)

Reply:
is the only reason for the subdirectory the `remove_definitions` call? setting/unsetting the variables can happen in the same scope AFAIK.

Reply:
The main reason was that I'd need to re-set Win32 definitions afterwards and thought moving to a subdirectory would be clearer. 
https://github.com/supercollider/supercollider/blob/c1b26f6458a4303453520f001256340f850996f0/external_libraries/CMakeLists.txt#L35

Also not sure what unsetting C/CXX flags does to the rest of the compilation (though it did work in that commit that I linked). 
Another minor thing is that `asiosdk` can now be in a subdirectory, and not at the same level as other modules, but not sure if that's worth anything.

Reply:
ok, that makes sense. i'm torn because i liked the directory structure being consistent, but it is a lot easier to understand with portaudio contained inside 2 subdirectories instead of 1. i think let's keep it this way and use the subdirectory names `portaudio/portaudio_submodule`, that's the clearest naming i can think of and it immediately explains the difference between the two. normally, i would suggest `portaudio/portaudio` but not if users are going to be navigating that hierarchy.